### PR TITLE
deploy individual initiatives

### PR DESCRIPTION
**Pull request by arjenhuitema**
Added documentation to deploy individual initiatives.

### CODE REVIEW

**Review by paulgrimley — APPROVED** [review]
Approved

File: docs/wiki/Deploy-individual-Policy-Initiatives.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+## Introduction
+The following guide describes the steps to implement an individual initiative. In this example we will deploy the Service Health initiative via Azure CLI. The same principles and steps apply to other initiatives and deployment methods. The deployment is similar to deploying all initiatives, and the main difference is in the initial deployment of the Policy Definitions. The high level steps are as follows:
+- You can choose deploy all the Policy Definitions, this is recommended when other initiatives will be implemented in the future.
```
Comment (paulgrimley) [inline docs/wiki/Deploy-individual-Policy-Initiatives.md:3]: i'm a little confused reading this and not clear on what is trying to be said, perhaps an example could help.

File: docs/wiki/Deploy-individual-Policy-Initiatives.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    "value": "true"
+                }
+            }
+        }
+    }
+}
+```
+
+## 3. Configuring variables for deployment
+Using either a PowerShell prompt or Azure CLI, navigate to the root of the cloned repo and log on to Azure with an account with at least Resource Policy Contributor access at the root of the management group hierarchy where you will be creating the policies and initiatives.
```
Comment (paulgrimley) [inline docs/wiki/Deploy-individual-Policy-Initiatives.md:74]:
```suggestion
Using either a PowerShell prompt or Azure CLI, navigate to the root of the cloned repo and log on to Azure with an account with at least [Resource Policy Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#resource-policy-contributor) access at the root of the management group hierarchy where you will be creating the policies and initiatives.
```

Wondering if we should add a link to this RBAC role - also wondering if this means we need to update other files where this has been copied from, can you check?

File: docs/wiki/Introduction-to-deploying-ALZ-Monitor.md [hunk 3]
```diff
@@ -138,4 +138,5 @@ In some scenarios, it may be necessary to remove everything deployed by the ALZ
 - To customize policy assignments, please proceed with [Customize Policy Assignment](./Customize-Policy-Assignment)
 - To deploy with GitHub Actions, please proceed with [Deploy with GitHub Actions](./Deploy-with-GitHub-Actions)
 - To deploy with Azure CLI, please proceed with [Deploy with Azure CLI](./Deploy-with-Azure-CLI)
-- To deploy with Azure PowerShell, please proceed with [Deploy with Azure PowerShell](./Deploy-with-Azure-PowerShell)
\ No newline at end of file
+- To deploy with Azure PowerShell, please proceed with [Deploy with Azure PowerShell](./Deploy-with-Azure-PowerShell)
+- To deploy an individual Policy Initiative, please proceed with [Deploy individual Policy Initiatives](./Deploy-individual-Policy-Initiatives)
```
Comment (paulgrimley) [inline docs/wiki/Introduction-to-deploying-ALZ-Monitor.md:142]:
```suggestion
-  To deploy Service Health or another Policy Initiative (individually), please proceed with [Deploy individual Policy Initiatives](./Deploy-individual-Policy-Initiatives)
```

File: docs/wiki/_Sidebar.md [hunk 4]
```diff
@@ -10,6 +10,7 @@
    * [Deploy with GitHub Actions](./Deploy-with-GitHub-Actions)
    * [Deploy with Azure CLI](./Deploy-with-Azure-CLI)
    * [Deploy with Azure PowerShell](./Deploy-with-Azure-PowerShell)
+   * [Deploy individual Policy Initiatives](./Deploy-individual-Policy-Initiatives)
```
Comment (paulgrimley) [inline docs/wiki/_Sidebar.md:13]:
```suggestion
   * [Deploy Service Health or individual Policy Initiatives](./Deploy-individual-Policy-Initiatives)
```

**Review by paulgrimley — CHANGES_REQUESTED** [review]
Requested some changes so we make sure Service Health is more prominent and also wanted some clarity on a couple of things

File: docs/wiki/Deploy-individual-Policy-Initiatives.md (hunk 2 above)
Comment (arjenhuitema) [reply docs/wiki/Deploy-individual-Policy-Initiatives.md:74]: Yes, if we change this, we should make the same changes in the other deployment guides.

File: docs/wiki/Deploy-individual-Policy-Initiatives.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
```
Comment (jfaurskov) [inline docs/wiki/Deploy-individual-Policy-Initiatives.md:84]: I feel we are muddying the waters a bit here. In all of our other documentation we have specified either Az CLI or PowerShell, but here we are mixing the two somewhat. Shouldn't we do one Az CLI and one PowerShell, clearly separated?

File: docs/wiki/Deploy-individual-Policy-Initiatives.md (hunk 2 above)
Comment (paulgrimley) [reply docs/wiki/Deploy-individual-Policy-Initiatives.md:84]: Are you able to update the other links as part of this PR Arjen please if it is not too much work

File: docs/wiki/Deploy-individual-Policy-Initiatives.md (hunk 5 above)
Comment (arjenhuitema) [reply docs/wiki/Deploy-individual-Policy-Initiatives.md:84]: This comes directly from our Azure CLI deployment guide. And the point here is not whether you use the Azure PowerShell deployment method or the Azure CLI deployment method. It´s about the command line tool preference. I do agree that our documentation can be clearer on this point, maybe we should rephrase this as:

> Open your preferred command line tool _(Windows PowerShell, Cmd, Bash or other Unix shells), and navigate to the root of the cloned repo and log on to Azure with an account with at least Resource Policy Contributor access at the root of the management group hierarchy where you will be creating the policies and initiatives.

Comment (arjenhuitema) [reply docs/wiki/Deploy-individual-Policy-Initiatives.md:84]: > Shouldn't we do one Az CLI and one PowerShell, clearly separated?

Sure, I can create a new page for a PowerShell specific deployment, I thought that it would be a bit redundant?

**Review by paulgrimley — DISMISSED** [review]
Thanks @arjenhuitema LGTM, @jfaurskov can you take a look?

File: docs/wiki/Introduction-to-deploying-ALZ-Monitor.md [hunk 6]
```diff
@@ -138,4 +138,6 @@ In some scenarios, it may be necessary to remove everything deployed by the ALZ
 - To customize policy assignments, please proceed with [Customize Policy Assignment](./Customize-Policy-Assignment)
 - To deploy with GitHub Actions, please proceed with [Deploy with GitHub Actions](./Deploy-with-GitHub-Actions)
 - To deploy with Azure CLI, please proceed with [Deploy with Azure CLI](./Deploy-with-Azure-CLI)
-- To deploy with Azure PowerShell, please proceed with [Deploy with Azure PowerShell](./Deploy-with-Azure-PowerShell)
\ No newline at end of file
+- To deploy with Azure PowerShell, please proceed with [Deploy with Azure PowerShell](./Deploy-with-Azure-PowerShell)
+- To deploy Service Health or another Policy Initiative (individually), please proceed with [Deploy individual Policy Initiatives with Azure CLI](./Deploy-individual-Policy-Initiatives-with-Azure-CLI)
+- To deploy Service Health or another Policy Initiative (individually), please proceed with [Deploy individual Policy Initiatives with Azure PowerShell](./Deploy-individual-Policy-Initiatives-with-Azure-PowerShell)
```
Comment (jfaurskov) [inline docs/wiki/Introduction-to-deploying-ALZ-Monitor.md:143]:
```suggestion
- To deploy Service Health or another Policy Initiative (individually) with PowerShell, please proceed with [Deploy individual Policy Initiatives with Azure PowerShell](./Deploy-individual-Policy-Initiatives-with-Azure-PowerShell)
```

File: docs/wiki/_Sidebar.md [hunk 7]
```diff
@@ -10,6 +10,8 @@
    * [Deploy with GitHub Actions](./Deploy-with-GitHub-Actions)
    * [Deploy with Azure CLI](./Deploy-with-Azure-CLI)
    * [Deploy with Azure PowerShell](./Deploy-with-Azure-PowerShell)
+   * [Deploy Service Health or individual Policy Initiatives with Azure CLI](./Deploy-individual-Policy-Initiatives-with-Azure-CLI)
```
Comment (jfaurskov) [inline docs/wiki/_Sidebar.md:13]:
```suggestion
   * [Deploy individual Policy Initiatives with Azure CLI](./Deploy-individual-Policy-Initiatives-with-Azure-CLI)
```

File: docs/wiki/_Sidebar.md [hunk 8]
```diff
@@ -10,6 +10,8 @@
...
+   * [Deploy Service Health or individual Policy Initiatives with Azure PowerShell](./Deploy-individual-Policy-Initiatives-with-Azure-PowerShell)
```
Comment (jfaurskov) [inline docs/wiki/_Sidebar.md:14]:
```suggestion
   * [Deploy individual Policy Initiatives with Azure PowerShell](./Deploy-individual-Policy-Initiatives-with-Azure-PowerShell)
```

**Review by jfaurskov — CHANGES_REQUESTED** [review]
Excellent work @arjenhuitema just a few suggestions. :-)

**Comment by arjenhuitema** [comment]
@jfaurskov great suggestions. Committed all.

**Review by jfaurskov — APPROVED** [review]
LGTM
